scripts/explicit_kwargs_checker_test.py

```diff
@@ -0,0 +1,77 @@
+# coding: utf-8
+#
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For details on how to write such tests, please refer to
+# https://github.com/oppia/oppia/wiki/Writing-Tests
+
+import os
+import sys
+import unittest
+
+_PARENT_DIR = os.path.abspath(os.path.join(os.getcwd(), os.pardir))
+_PYLINT_PATH = os.path.join(_PARENT_DIR, 'oppia_tools', 'pylint-1.7.1')
+sys.path.insert(0, _PYLINT_PATH)
+
+# Since this module needs to be imported after adding Isort path,
```

Comment:
I don't understand what's going on here. Could you elaborate/rephrase the comment?

Reply:
Since we are placing not placing the modules in alphabetical order, it results in an import-order error. I'll rephrase the comment to something more clear.

Reply:
But these are third-party modules (i.e. not python stdlib). Third-party modules usually do get placed in a separate block below the stdlibs. So why is this an issue for the linter?

Reply:
Sorry! The problem here is we cannot have any statements between the module imports. Therefore I had to skip isort for these lines.

Reply:
OK! That explanation makes more sense.